dev/mcal/inc/stm32f103xx.h

```diff
@@ -82,6 +82,79 @@ struct RccRegDef{
 #define RCC  (reinterpret_cast<volatile RccRegDef*>(RCC_BASE_ADDRESS))
 
 }  // namespace rcc
+
+namespace gpio {
+struct GpioRegDef {
+    RegWidth_t CRL;     // Port configuration register low
+    RegWidth_t CRH;     // Port configuration register High
```

Comment:
Add a descrption for each Register
I would suggest using chatGPT instead of writing messages manually. It can provide automated assistance and save you time.se chatGPT instead of writing it by yourself